rules/when-you-reply-inline-do-you-use-a-different-color/rule.md

```diff
@@ -40,25 +42,71 @@ I have replied inline
 Regards,
 Daniel
 
------Original Message-----
-From: Adam Cogan www.ssw.com.au
-Sent: Monday, 27 May 2002 8:00 AM
-To: Daniel Hyles www.ssw.com.au 
-Subject: Daniel Hyles - Morning Goals
+-----Original Message-----   
+**From:** Adam Cogan www.ssw.com.au   
+**Sent:** Monday, 27 May 2002 8:00 AM   
```

Comment:
Should we make the date more modern (aka 2021)? Also, I have the same question for any emails in this rule.

Reply:
agreed and done